package.json

```
@@ -9,7 +9,7 @@
     "@webassemblyjs/helper-module-context": "1.9.0",
     "@webassemblyjs/wasm-edit": "1.9.0",
     "@webassemblyjs/wasm-parser": "1.9.0",
-    "acorn": "^6.2.1",
+    "acorn": "^7.1.1",
```

```suggestion
    "acorn": "^6.4.1",
```
[Version 6.4.1 has been released with the same fix as 7.1.1](https://github.com/webpack/webpack/pull/10518#issuecomment-596467132), use that version to prevent a breaking change.